rafs/src/metadata/layout/v6.rs

```diff
@@ -177,7 +179,7 @@ impl RafsV6SuperBlock {
             return Err(einval!("invalid extended slots in Rafsv6 superblock"));
         }
 
-        if self.s_blocks == 0 {
+        if u32::from_le(self.s_blocks) == 0 {
```

Comment:
as a generic rule of linux, no need to convert endian when comparing with 0

Reply:
Fixed several similar cases